pkg/cri/config/config.go

```diff
@@ -319,8 +319,6 @@ type PluginConfig struct {
 	ContainerdConfig `toml:"containerd" json:"containerd"`
 	// CniConfig contains config related to cni
 	CniConfig `toml:"cni" json:"cni"`
-	// DisableTCPService disables serving CRI on the TCP server.
-	DisableTCPService bool `toml:"disable_tcp_service" json:"disableTCPService"`
```

Comment:
This removal is a breaking change and needs to be documented in RELEASES.md

Reply:
It's not removed, its just not moved over with the rest of the runtime configurations under `io.containerd.cri.v1.runtime`. It will stay at the original location under `io.containerd.grpc.v1.cri`. We will need to update our same configurations once the final 2.0 configuration is finalized.